### PR TITLE
Add SwiftUI sample stub

### PR DESCRIPTION
**Pull request by cardoso**
No Stream Chat logic in it yet. Just some basic UI elements to start.

#no_changelog

### CODE REVIEW

File: Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import StreamChatClient
+
+@available(iOS 13, *)
+struct ChatView: View {
+    @State
+    var channelId: String
+    @State
+    var text: String = ""
+    @State
+    var messages: [String] = []
```
Comment (VojtaStavik) [inline Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift:17]: lol the formatter :D Should be fixed now, right @buh ?

**Review by VojtaStavik — APPROVED** [review]
Thank you for such a quick PR!

File: Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift (hunk 1 above)
Comment (buh) [reply Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift:17]: This is I explicitly force to write on the same line. I no idea why it's here.
But yes it should be fixed.

Comment (cardoso) [reply Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift]: I ran the formatter manually and pushed a fix. Thought it would do it on pre-commit

**Comment by VojtaStavik** [comment]
@cardoso please squash when merging ;-)

**Pull request merged**